pkg/endpoint/bpf.go

```diff
@@ -466,7 +466,9 @@ func (e *Endpoint) removeOldRedirects(owner Owner, desiredRedirects map[string]b
 // Must be called with endpoint.Mutex not held and endpoint.BuildMutex held.
 // Returns the policy revision number when the regeneration has called, a
 // boolean if the BPF compilation was executed and an error in case of an error.
-func (e *Endpoint) regenerateBPF(owner Owner, currentDir, nextDir string, regenContext *RegenerationContext) (revnum uint64, compiled bool, reterr error) {
+// Also returns a function which performs additional operations depending on
+// the state of the returned error.
```

Comment:
The comment must say that the function must be invoked when it is non-nil.

Reply:
Done.